docs/projects/automate-project-git-source.ipynb

```diff
@@ -0,0 +1,2069 @@
+{
```

Comment:
<p>"# load from a private repo" - can delete this line</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/3640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
deleted